src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -45,13 +45,6 @@
 #import "JNIUtilities.h"
 #import "AWTView.h"
 
-
-// these constants are duplicated in CAccessibility.java
-#define JAVA_AX_ALL_CHILDREN (-1)
-#define JAVA_AX_SELECTED_CHILDREN (-2)
-#define JAVA_AX_VISIBLE_CHILDREN (-3)
-// If the value is >=0, it's an index
-
```

Comment:
Since we are going to get rid of the JavaComponentAccessibility in the future i would rather copy the definitions into the CommonComponentAccessibility.h - that would save the double work in the future.

Reply:
done